docs/reference/release-notes/8.0.0-rc2.asciidoc

```diff
@@ -15,6 +15,27 @@ data loss. If you upgrade from a released version, such as 7.16, to a
 pre-release version for testing, discard the contents of the cluster when you are
 done. Do not attempt to upgrade to the final 8.0 release.
 
+* If you installed {es} from an archive on Linux aarch64 or macOS aarch64, the
```

Comment:
Do we think readers understand `aarch64`? Do we need to explicitly say "ARM" or "M1"?

Reply:
Good point, I used Linux aarch64 and macOS aarch64 because that's what the download option says in the download page but we can definitely change this to "Linux ARM or macOS M1"